packages/ckeditor5-table/tests/_utils/utils.js

```diff
@@ -32,19 +32,26 @@ const WIDGET_TABLE_CELL_CLASS = 'ck-editor__editable ck-editor__nested-editable'
  *		};
  *
  * @param {Array.<Array.<String>|Object>} tableData
- * @param {Object} [attributes] Optional table attributes: `headingRows` and `headingColumns`.
+ * @param {Object} [attributes={}] Optional table attributes.
+ * @param {Number} attributes.headingRows Number of heading rows.
+ * @param {Number} attributes.headingColumns Number of heading columns.
+ * @param {Object} attributes.defaultProperties Default styles for the table.
+ * @param {Object} attributes.defaultCellProperties Default styles for all cells.
```

Comment:
It's probably more like `options` than `attributes` in that case.

Reply:
Updated in https://github.com/ckeditor/ckeditor5/pull/9393/commits/88046d8cf94998dd3bc40c7ffc6317c2676fc369.